migtests/tests/analyze-schema/dummy-export-dir/schema/tables/table.sql

```diff
@@ -229,5 +237,3 @@ CREATE TABLE uritype_test (
 ) ;
 -- valid
 Alter table only parent_tbl add constraint party_profile_pk primary key (party_profile_id);
--- alter table not supported
```

Comment:
Since we are planning to report these under unsupported PG syntax, should we keep them in the test, and assert?

Reply:
yeah we can a few of them and assert